src/Symfony/Component/VarDumper/Caster/IntlCaster.php

```diff
@@ -34,4 +34,79 @@ public static function castMessageFormatter(\MessageFormatter $c, array $a, Stub
 
         return $a;
     }
+
+    public static function castNumberFormatter(\NumberFormatter $c, array $a, Stub $stub, $isNested)
+    {
+        $a += array(
+            Caster::PREFIX_VIRTUAL.'locale' => $c->getLocale(),
+            Caster::PREFIX_VIRTUAL.'pattern' => $c->getPattern(),
+            Caster::PREFIX_VIRTUAL.'attributes' => new EnumStub(
+                array(
+                    'PARSE_INT_ONLY' => $c->getSymbol(\NumberFormatter::PARSE_INT_ONLY),
```

Comment:
shouldn't this use getAttribute instead?

Reply:
Yes it should. It is fixed now.